setup.cfg

```diff
@@ -28,7 +28,7 @@ classifier =
 
 [options]
 zip_safe = true
-python_requires = >=3.7,!=3.0,!=3.1,!=3.2,!=3.3
```

Comment:
Wouldn't just `python_requires = >=3.7` be sufficient?

Reply:
Probably, I just went along with the old convention.